vitals/BatteryFactorsStateEstimator.java

```diff
@@ -0,0 +1,84 @@
+package vitals;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BatteryFactorsStateEstimator {

```

Comment:
BatteryFactorsStateEstimator holds too many functionalities

why is********* series function invoke breach functionalities